src/Parser/Monk/Windwalker/Modules/Spells/TouchOfDeath.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Enemies from 'Parser/Core/Modules/Enemies';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import StatTracker from 'Parser/Core/Modules/StatTracker';
+
+import SpellIcon from 'common/SpellIcon';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import AbilityTracker from 'Parser/Core/Modules/AbilityTracker';
+
+class TouchOfDeath extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+    enemies: Enemies,
+    statTracker: StatTracker,
+    abilityTracker: AbilityTracker,
+  };
+
+  expectedBaseDamage = 0;
+  expectedGaleBurst = 0;
+  totalGaleBurst = 0;
+  highestGaleBurst = 0;
+  // Vulnerability amplifiers are target specific damage taken increases like seen on Kin'garoth adds. 
+  highestVulnerabilityAmplifier = 0;
+  totalVulnerabilityAmplifier = 0;
+
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+    if (SPELLS.TOUCH_OF_DEATH.id !== spellId) {
+      return;
+    }
+    this.expectedGaleBurst = 0;
+    const masteryPercentage = this.statTracker.currentMasteryPercentage;
+    const versatilityPercentage = this.statTracker.currentVersatilityPercentage;
+    this.expectedBaseDamage = event.maxHitPoints * 0.5 * (1 + masteryPercentage) * (1 + versatilityPercentage);
```

Comment:
What's the 0.5? Should make it a constant so it has a name. Same with the 0.3 and 0.1 SEF multipliers below.

Reply:
Touch of Deaths base damage is half of the casters max health, amplified by mastery and versatility. Gale burst is 10% of damage done to the target with Touch of Death, but as mentioned in the comment, is raised to 30% while SEF is active. I can move the 0.5 to be a constant, although i feel that 'HP*0.5' is pretty self-explanatory about what it does. The Gale Burst 0.1 could be moved to a constant and then the SEF multiplier would be either 1x or 3x that.